components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -62,6 +66,10 @@ internal class MediaSessionServiceDelegate(
         mediaSession.setCallback(MediaSessionCallback(store))
         notificationScope = MainScope()
 
+        runBlocking {
+            startForegroundNotification()
+        }
```

Comment:
Mh, this is dangerous. If we actively block the main thread here then we end up freezing the app and/or increase the risk of not calling startForeground fast enoug? 🤔

Reply:
I've added a new function so this call is not to suspend function anymore.  Thanks,